ludwig/model_export/onnx_exporter.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC, abstractmethod
+
+import torch
+
+from ludwig.api import LudwigModel
+
+
+class OnnxExporter(ABC):
+    def __init__(self, model):
+        self.model = model
+
+    @abstractmethod
+    def forward(self, x):
+        return self.model({"image_path": x})
+
+    def export_classifier(self, model_path, export_path):
+        ludwig_model = LudwigModel.load(model_path)
+        model = OnnxExporter(ludwig_model.model)  # Wrap the model
+        model.eval()  # inference mode, is this needed.. I think onnx export does this for us
```

Comment:
I think the `ONNX` exporter does the `eval()`. Can you double check?

Reply:
removed